src/modules/storage.js

```diff
@@ -1,14 +1,34 @@
+import update from './updateDom.js';
+
 class Scores {
   constructor() {
     this.scores = [];
   }
 
   add(name, score) {
```

Comment:
- [ ] it would be better if you use the `Arrow function` instead of `normal function` in order to practice what you've learned and also you will be familiar with both